.cascade-code/Chapman.edu/_components/omni_nav_v2/omni_nav.vtl

```diff
@@ -318,7 +319,9 @@
 
 #macro ( buildPrimaryNavSearchBox )
   <div class="nav-container primary-nav-search">
-    #buildSearchBox()
+    <div class="cu-two-column-gcs" id ="primary-nav-search">
```

Comment:
You've got an extra space in `id ="primary-nav-search"`.

More importantly, does this need an `id`? Having a child block with `id` that is same as parent's `class` name doesn't smell right.

Reply:
Each search element needs to have a unique identifier for GCS to differentiate them so I thought it made sense to use an `id`. I should be able to refactor this to only have `id="primary-nav-search"` and remove the redundant `class`.